src/lib/containers/personal_access_token/AccessTokenPage.tsx

```diff
@@ -18,53 +15,23 @@ export const AccessTokenPage: React.FunctionComponent<AccessTokenPageProps> = ({
   title,
   body,
 }: AccessTokenPageProps) => {
-  const { accessToken } = useSynapseContext()
-  const [isLoading, setIsLoading] = useState(false)
-
   const [showCreateTokenModal, setShowCreateTokenModal] = useState(false)
 
   const {
```

Comment:
This test suite was failing with logs after tests were done. Since those are often caused by state updates triggered after fetch, let's just replace this with a react-query hook, which handles that scenario internally. Also has the benefit of cleaning up a lot of this code. Test suite passed without any changes.